README.md

```diff
@@ -51,31 +51,31 @@ SiteCatalyst.push({
 
 `SiteCatalyst.push` requires an object to be passed with attributes defining the type of event. The following options are available.
 
-| Parameter   | Options                   | Required |
-|-------------|---------------------------|----------|
-| type        | 'load', 'event'           |     -    |
-| clickAction | 'action', 'link'          |     -    |
-| linkType    | 'o', 'e', 'd'             |     -    |
-| linkName    | 'Download PDF'            |     -    |
-| pageName    | 'Page name value'         |     -    |
-| campaign    | 'Your campaign ID'        |     -    |
-| channel     | 'Channel Goes Here'       |     -    |
-| eVar        | 'eVar Value (e.g. eVar3)' |     -    |
-| events      | 'event2 to event3'        |     -    |
-| hier        | 'Hier string'             |     -    |
-| prop        | 'prop value (e.g. prop2)' |     -    |
-| products    | 'Product values go here'  |     -    |
-| server      | 'UK Site'                 |     -    |
-
-### Links
-
-There are different link types (`linkType`).
-
-| Option | Description |
-|--------|-------------|
-| o      | other       |
-| e      | exit link   |
-| e      | download    |
+| Parameter     | Options                   | Required |
+|---------------|---------------------------|----------|
+| `type`        | 'load', 'event'           |     -    |
+| `clickAction` | 'action', 'link'          |     -    |
+| `linkType`    | 'o', 'e', 'd'             |     -    |
+| `linkName`    | 'Download PDF'            |     -    |
+| `pageName`    | 'Page name value'         |     -    |
+| `campaign`    | 'Your campaign ID'        |     -    |
+| `channel`     | 'Channel Goes Here'       |     -    |
+| `eVar`        | 'eVar Value (e.g. eVar3)' |     -    |
+| `events`      | 'event2 to event3'        |     -    |
```

Comment:
This is more in relation to the previous MR that introduced this change.

I do think this description could be a little confusing, as events can be commas separated event values. 
EG, event2,event5,event21.